src/test/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannelTest.java

```diff
@@ -435,8 +435,7 @@ public void testInsertRows_GetOffsetTokenFailureAfterReopenChannel() throws Exce
       TopicPartitionChannel.StreamingBuffer streamingBuffer =
           topicPartitionChannel.new StreamingBuffer();
       streamingBuffer.insert(records.get(0));
-      InsertValidationResponse response =
-          topicPartitionChannel.insertBufferedRecords(streamingBuffer);
+      topicPartitionChannel.insertBufferedRecordsWithRetry(streamingBuffer, 1);
```

Comment:
not major: To avoid so many changes in all files you can instead do this:
Recreate private insertBufferedRecordsWithRetry(streamingBuffer, 1) instead of adding more parameters into the function and call this newly created method from the existing method. 

so insertBufferedRecords(buffer) calls insertBufferedRecirds(buffer, 1)
and tests can continue to call insertBufferedRecords(buffer)

Reply:
lol, actually that was exactly what I've done several commits ago, but Toby thinks this creates extra redundancy, so it is changed to what it is now.

Reply:
I see, okay